tools/ci/build.sh

```diff
@@ -10,13 +10,12 @@ CMAKE_FLAGS="${2}"
 
```

Comment:
why not put these to tools/jenkins/... so its clear where its used?

Reply:
Used by all CI